src/pages/patientView/mutation/column/AlleleFreqColumnFormatter.tsx

```diff
@@ -161,18 +153,37 @@ export default class AlleleFreqColumnFormatter {
 
 
     public static getFrequency(data:Mutation[]): string|string[] {
-        let result = [];
+        const result: string[] = [];
+
         if (data) {
-            for (let mutation of data) {
-                let frequency = mutation.tumorAltCount/(mutation.tumorAltCount+mutation.tumorRefCount);
-                if (frequency) {
```

Comment:
This was filtering out zero values like `0.0`, that's why we were ending up with an smaller array size and getting the error message `Not all the arrays returned from the download functions are from the same length.`